src/demo/app/datepicker/datepicker-demo.component.ts

```diff
@@ -13,7 +13,14 @@ import * as moment from 'moment';
       </dl>
       <ul>
         <li>
-          <code>@Input() date: Date</code> (optional) bound input to sets datepicker date
+          <code>@Input() format: string</code> (optional) sets the datepicker format shown in the form field,
```

Comment:
should we provide any guidance for what formats are acceptable? or is `moment` pretty forgiving?